qualtran/surface_code/algorithm_summary.py

```diff
@@ -41,3 +41,21 @@ class AlgorithmSummary:
     toffoli_gates: float = _PRETTY_FLOAT
     rotation_gates: float = _PRETTY_FLOAT
     rotation_circuit_depth: float = _PRETTY_FLOAT
+
+    def __mul__(self, other: int) -> 'AlgorithmSummary':
```

Comment:
is it weird that this supports multiplication but not other arithmetic?

Reply:
I implemented addition and subtraction as well. I don't think division make sense here